packages/scripts/import/parse-csv.ts

```diff
@@ -1,14 +1,29 @@
 import { readFileSync } from 'fs';
 import { parse } from 'csv-parse/sync';
 
-export function parseCSVFrom(file: string): Record<string, any>[] {
-  return parse(file, {
+export function parseCSVFrom(contents: string): Record<string, any>[] {
+  return parse(contents, {
     columns: true,
     skip_empty_lines: true,
+    trim: true,
```

Comment:
@Danble this is all that's needed. https://csv.js.org/parse/options/trim/

It's easy because the tool already provides that option, but if it hadn't provided it, new code should be placed into parseCSVFrom and not convertJsonRowToEntryFormat since trimming whitespace is part of parsing and not converting.

Also, keep your tests simpler, note how an extremely small test can check whitespace in mutliple header and content row contexts.